src/librustc/session/config.rs

```diff
@@ -709,6 +709,36 @@ pub fn append_configuration(cfg: &mut ast::CrateConfig,
     }
 }
 
+unsafe fn configure_llvm(sess: &Session) {
```

Comment:
If one doesn't exist, can one be made? I'd be fine just documenting that things won't work unless some initialization function is called first.